cf-serverd/server_transform.c

```diff
@@ -231,6 +231,8 @@ void KeepPromises(EvalContext *ctx, const Policy *policy, GenericAgentConfig *co
     roles_acl     = calloc(1, sizeof(*roles_acl));
     SV.path_shortcuts = StringMapNew();
 
+BREAKPOINT_KEEP_PROMISES_FAIL: // see tests/acceptance/16_cf-serverd/keep_promises_fail.cf
+
```

Comment:
for what it's worth... I tried some 

``
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wunused-label"
...
#pragma GCC diagnostic pop
``

But it didn't seem to remove the warnings when I was compiling locally. :(

Reply:
as it turns out these unused labels don't seem to break travis/jenkins even though -Werror seems to be enabled. I fixed cf-monitor/env_monitor.c label with a gcc Label Attribute like:
```
__attribute__((unused));
```

So maybe just to be safe I will add that to these labels as well.